test/fixtures/empty-config/eslint.config.mjs

```diff
@@ -0,0 +1 @@
+export default [{}];
```

Comment:
Oh, I was imagine no `eslint.config` at all here, or that would be a new case?

Reply:
When I proposed that I wasn't able to reproduce it yet, so I didn't know what problem I was facing. Upon giving it more thought it makes more sense to me that an empty configuration would yield success because that's what an empty legacy config does as well, and a missing configuration will cause ESLint itself to fail with the error that no configuration has been found, so we'll still get that one.